src/views/gallery.hbs

```diff
@@ -13,12 +13,13 @@
 {{/section}}
 
 <div class="container">
-  <div id="pixlee_container"></div>
-  <script type="text/javascript">
-    window.PixleeAsyncInit = function () {
-      Pixlee.init({ apiKey: "{{apiKey}}" });
-      Pixlee.addSimpleWidget({ widgetId: "{{widgetID}}" });
-    };
-  </script>
-  <script src="//instafeed.assets.pixlee.com/assets/pixlee_widget_1_0_0.js"></script>
+  <!-- LightWidget WIDGET -->
+  <script src="https://cdn.lightwidget.com/widgets/lightwidget.js"></script>
+  <iframe
+    src="//lightwidget.com/widgets/0d77edd5936c59389ff1626d5aefb1a2.html"
```

Comment:
two questions:
- Does it work with https? Looks like it doesn't 
<img width="1062" alt="image" src="https://github.com/Lernwerkstatt/site/assets/15848876/fb55e8f0-50ca-4fdf-a302-dd6a5a898aaf">

- No keys required? Is that already our unique ID?

Reply:
The free version does not work with https but it would be only a one time fee of 10$ to upgrade it. I was going to pay that.

No keys required, the ID is unique. Would be better to set it as environment variable, good point there. If you are okay with the tool I'll create a new widget with a new id that's not publicly visible.

Reply:
I activated a 14 day free trial so it works with https. If everything works I'll upgrade to the paid version.